### PR TITLE
Update getNodeTime.yml

### PR DESCRIPTION
**Pull request by zachamo**
Update to description field

### CODE REVIEW

**Comment by zachamo** [comment]
Drats - this content is already embedded in the response description.. Neglected to drill down into the data array in the documentation.

**Comment by segfaultxavi** [comment]
I also find the current REST docs a bit hard to parse. There's an open issue to improve this: https://github.com/symbol/symbol-docs/issues/588

Anyway, thanks for your interest!
